packages/components/src/app-promo-card/index.tsx

```diff
@@ -87,7 +83,7 @@ export default function AppPromoFullWidth( {
 					components: {
 						a: (
 							<a
-								className="woo"
+								className="app-promo-card__woo-link"
```

Comment:
SCSS applies additional styling for `.app-promo-card__link` but the general link class is not added here and below for jetpack.

Reply:
Ah, good catch. Fixed!